src/spec-configuration/containerCollectionsOCI.ts

```diff
@@ -28,7 +29,10 @@ export interface OCIRef {
 	path: string;			// 'devcontainers/features/go'
 	resource: string;		// 'ghcr.io/devcontainers/features/go'
 	id: string;				// 'go'
-	version?: string;		// '1.0.0'
+
+	version: string;		// (Either the contents of 'tag' or 'digest')
```

Comment:
Would it clarify the code if you omit the `version` property? I guess tags and digests cannot always be used the same way and having (only) separate properties makes it less likely to overlook the other case when writing and reviewing code?

Reply:
Actually the registry api handles both tag and digest the same way, so it's cleaner in a lot of the codebase to have a single 'version' that is interpolated into the request paths.  I added both `tag` and `digest` in case there are instances in the codespace where it's interesting to know which is which.